src/plugins/index.js

```diff
@@ -36,6 +37,42 @@ export default class Plugins {
     this.lock = new Lock(this.out)
   }
 
+  clearCache (...paths: string[]) {
+    this.cache.deletePlugin(...paths)
+  }
+
+  async _installCheck (name: string, tag: string) {
```

Comment:
@dickeyxxx removed the class split and used flag instead in https://github.com/heroku/cli-engine/pull/211/commits/842480dc3b7fca40165a482c493c6f4266ded28e